### PR TITLE
Chore: use `Box` in command palette empty state

### PR DESCRIPTION
**Pull request by ashharrison90**
**What is this feature?**

- uses `Box` instead of a wrapping `div`

**Why do we need this feature?**

- nicer 😄 

**Who is this feature for?**

- grafana devs

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

**Review by JoaoSilvaGrafana — APPROVED** [review]
👏

File: public/app/features/commandPalette/EmptyState.tsx [hunk 1]
```diff
@@ -1,31 +1,21 @@
...
 import { Trans } from 'app/core/internationalization';
 
 export interface Props {}
 
 export const EmptyState = ({}: Props) => {
-  const styles = useStyles2(getStyles);
-
   return (
-    <div className={styles.container}>
+    <Box paddingTop={8} paddingBottom={8}>
```
Comment (JoaoSilvaGrafana) [inline public/app/features/commandPalette/EmptyState.tsx:10]:
```suggestion
    <Box paddingY={8}>
```

**Review by JoaoSilvaGrafana — CHANGES_REQUESTED** [review]
Requesting changes so it doesnt automerge 😅

File: public/app/features/commandPalette/EmptyState.tsx (hunk 1 above)
Comment (ashharrison90) [reply public/app/features/commandPalette/EmptyState.tsx:10]: ahhhh nice ty 👍

**Review by JoaoSilvaGrafana — APPROVED** [review]
🙏

**Pull request merged**